deploy/crds/smartgateway.infra.watch_smartgateways_crd.yaml

```diff
@@ -79,24 +79,17 @@ spec:
             exporterPort:
               description: Metrics port for Prometheus to export. Defaults to 8081.
               type: integer
-            cpuStats:
```

Comment:
I don't think I ever tried this in the old code.

Reply:
Yeah it's currently unused anyway. There's a cpustats option, it just isn't served by an http server